stripe.go

```diff
@@ -78,6 +79,7 @@ type BackendConfiguration struct {
 	Type       SupportedBackend
 	URL        string
 	HTTPClient *http.Client
+	sync.Mutex
```

Comment:
Let's give this a name instead of embedding the struct. `mu` is usually a good candidate and used elsewhere in the code here.

Reply:
Will do. 

Also, some of the code that looks like I formatted it is just artifacts of my first (naiive) iteration. I can change those back if you wish -- apologies for the "larger than it needs to be" PR.

Reply:
Maybe hold off just a sec — I'm a little tied up right now, but was in progress of writing a slightly more comprehensive comment on here.

Reply:
:-X Pushed before I saw this.